mojap_metadata/logging_functions.py

```diff
@@ -0,0 +1,33 @@
+import logging
```

Comment:
I don't think we need logging?

Reply:
it is/was for the decorator, although a bit overkill yeah. will leave it in until it is removed when I have done the glue converter update